lib/mayaHydra/hydraExtensions/sceneIndex/mhMayaUsdProxyShapeSceneIndex.h

```diff
@@ -99,6 +99,11 @@ class MayaUsdProxyShapeSceneIndex : public HdSingleInputFilteringSceneIndexBase
         _SendPrimsDirtied(entries);
     }
 
+#ifndef CODE_COVERAGE_WORKAROUND
+private:
+#endif
+    void _Destroy();
```

Comment:
Add Destroy() method to perform proper cleanup on code coverage builds.

Reply:
Do we have a ticket to track when to remove the CODE_COVERAGE_WORKAROUND?

Reply:
Just created one: HYDRA-1297